javascripts/configureImportNotification.js

```diff
@@ -0,0 +1,47 @@
+(function () {
+  var notificationID = 'ConfigureGAImportNotification';
+  document.addEventListener("DOMContentLoaded", function (event) {
+    window.CoreHome.Matomo.on("matomoPageChange", showNotification);
+    window.CoreHome.Matomo.on("piwikPageChange", showNotification);
+  });
+  checkForConfigureImporterDisplay();
+
+  function checkForConfigureImporterDisplay() {
+    console.log('calling..');
+    if (!piwik.idSite) {
+      return;
+    }
+    console.log('calling1..');
```

Comment:
Looks like you've got a couple `console.log()` calls in here. Were those supposed to be removed after debugging?

Reply:
@snake14 Thanks, it was debugging and forgot to remove it,its removed now